lib/views/after_auth_screens/feed/individual_post.dart

```diff
@@ -193,7 +194,8 @@ class CommentTemplate extends StatelessWidget {
     return Row(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        const CircleAvatar(),
+        CircleAvatar(
+            backgroundImage: NetworkImage(comment.creator?.image ?? "")),
         Expanded(
```

Comment:
Please also add a fallback image to this as well

Reply:
will do it, thanks!